hypha/apply/users/templates/users/register.html

```diff
@@ -21,5 +21,8 @@
         {% endfor %}
         <button class="link link--button-secondary" type="submit">{% trans "Register" %}</button>
     </form>
+    <p>
+        Already have an account? <a href="{% url 'users_public:login' %}">Log in</a>
```

Comment:
Good addition, please make the strings translatable and then we can merge it in.

Reply:
Nice catch! Added :)